src/mixins/dropzone_mixin.js

```diff
@@ -64,10 +66,11 @@ const DropzoneMixin = {
   },
 
   onAccept: AttachmentActions.uploadAttachment,
+
   onSending(file, xhr, formData) {
```

Comment:
You've explained in your comment on Github what this function does, but adding a comment in the code might be useful for readability's sake.